docs/management_api.yml

```diff
@@ -102,7 +102,45 @@ paths:
           description: ID of the device to query.
       responses:
         204:
-          description: Created
+          description: Success
+        400:
+          description: Bad Request.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
```

Comment:
to me, it looks like the endpoint can also return 404 not found

Reply:
right, added, thanks!